### ISSUE TITLE
"Supply the arsonist" mission log asks for more items than what is required to complete the quest

### ISSUE THREAD

**Issue by tsulh** [CleverRaven/Cataclysm-DDA#48145]
# Describe the bug

"Supply the arsonist" mission asks to bring 42 kg of commercial fertilizer. However, Arsonist took only 2 sacks of commercial fertilizer from me, which totals to 4.08 kg, or 120 portions of commercial fertilizer. 

# Steps To Reproduce

Take quest "supply the Arsonist" at the refugee center and bring 120 portions of commercial fertilizer to Arsonist. It should be enough to finish the quest.

# Expected behavior

The same amount of commercial fertilizer is stated in the quest log and consumed during quest completion.

# Screenshots

![image](https://user-images.githubusercontent.com/6495727/111858103-f6693600-893e-11eb-82c1-9715470ccb0f.png)

# Versions and configuration

- OS: Linux
    - OS Version: Distributor ID: Debian; Description: Debian GNU/Linux 10 (buster); Release: 10; Codename: buster; 
- Game Version: 0.E-10118-g4338567d94-dirty [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Stats Through Skills [StatsThroughSkills]
]

# Additional context

The extra code in my game is to test a pet neutering feature, it doesn't touch the quests. Only these 4 files are edited: src/monster.cpp, src/monster.h, src/monexamine.cpp, src/monexamine.h.

**Comment by tsulh** [comment]
[save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6174887/save.zip)
I recalled that I have save from the same day when I completed the quest but before the completion, could be useful for confirmation.
The required fertilizer should be in the trunk of the small car with spikes. The arsonist is in the refugee center. Walk past the first room, then north along the corridor until reaching the door to the west.
